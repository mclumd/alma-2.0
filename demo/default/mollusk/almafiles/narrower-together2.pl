rel(mollusk, manny).
rel(cephalopod, steve).
rel(nautilus, nancy).
