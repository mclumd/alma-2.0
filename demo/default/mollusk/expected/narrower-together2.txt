rel(mollusk, manny)
rel(shell_bearer, manny)
rel(cephalopod, steve)
rel(mollusk, steve)
abnormal(steve, mollusk, "~rel(shell_bearer, steve)")
~rel(shell_bearer, steve)
rel(nautilus, nancy)
rel(cephalopod, nancy)
rel(mollusk, nancy)
abnormal(nancy, cephalopod, "rel(shell_bearer, nancy)")
rel(shell_bearer, nancy)
